.drone/drone.jsonnet

```diff
@@ -92,6 +98,33 @@ local docker_manifest(app) = {
   },
 };
 
+local deploy_to_dev() = {
+  image: 'us.gcr.io/kubernetes-dev/drone/plugins/updater',
+  settings: {
+    config_json: |||
+        {
+          "destination_branch": "master",
+          "pull_request_branch_prefix": "cd-tempo-dev",
+          "pull_request_enabled": false,
+          "pull_request_team_reviewers": [
+            "tempo-squad"
+          ],
+          "repo_name": "deployment_tools",
+          "update_jsonnet_attribute_configs": [
+            {
+              "file_path": "ksonnet/environments/tempo/dev-us-central-0.tempo-dev-01/images.libsonnet",
+              "jsonnet_key": "tempo",
```

Comment:
This will only update the Tempo image, can we also update `tempo_query` and `tempo_vulture`? I don't expect breaking changes between those but the whole process will be more consistent.

Reply:
Hmm, I think this is a limitation of the updater, it only takes one string in `jsonnet_key`. Not sure if @jvrplmlmn knows a way around this.

Reply:
@kvrhdn has let me know that we can use multiple `update_jsonnet_attribute_configs` blocks. So will add that here.